### ISSUE TITLE
TrackNet V3 Training

### ISSUE THREAD

**Issue by Zain-mehmood001** [alenzenx/TrackNetV3#12]
### Dataset

The badminton dataset annotated by author is being used for training

### Environment

Running on **NVIDIA RTX A4000**
Python version **3.9.13**
Torch version **2.0.1+cu117**
CUDA version **11.7**
CUnn version **8.9.7**

### Issue

The accuracy stays at 6.3% with 0 precision 0 Recall. 

### Screenshots

![Training Information](https://github.com/user-attachments/assets/6a1bbae5-49bb-44ce-bbc3-5267b3f708f1)

**Comment by alenzenx** [comment]
Hello, the data set must go through zz_Tracknet_badminton_DataConvert.py twice before training, once for training set conversion, and once for test set conversion. Finally, you need to run preprocess.py to completely convert the format, and finally you can start training. Please check if there is any No steps were missed.

**Comment by 1xxjie** [comment]
> 您好，数据集在训练前必须经过两次zz_Tracknet_badminton_DataConvert.py，一次用于训练集转换，一次用于测试集转换。最后，您需要运行 preprocess.py 以完全转换格式，最后您可以开始训练。请检查是否有任何 No steps were miss。

I've done it all, but it's still the same as this picture 

![Image](https://github.com/user-attachments/assets/bf297cb4-1deb-427d-9363-996ab6df209e)

**Comment by alenzenx** [comment]
好像只有NVIDIA RTX A4000會出現這個問題，
這是ChatGPT的回答 : 
A4000 可能因數據不平衡、CUDA/cuDNN 版本、混合精度計算導致模型無法學習。
先檢查數據分佈與 batch size，再測試不同的損失函數與 AMP 設置。
如果 3060 正常，而 A4000 失敗，可以嘗試 更新 PyTorch & CUDA，或使用 FP32 訓練。
可以先試試這些方法，看看能否改善！🚀

**Comment by alenzenx** [comment]
我的建議是試試修改 : 混合精度計算、batch size。我的手上沒有A4000只能請您試試了。

**Comment by Zain-mehmood001** [comment]
Hi there!

I fixed it by increasing the training set. I suggest keeping a 90% Training Set and 10% Testing/Validation Set
